DuckDuckGo/AppDelegate.swift

```diff
@@ -82,13 +82,22 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
     
     private func startOnboardingFlowIfNotSeenBefore() {
+        
         var settings = TutorialSettings()
         if !settings.hasSeenOnboarding {
+            startMigration()
```

Comment:
It deletes the old searches and favourites so that they don't get migrated more than once, but doesn't do anything to the bookmarks in the new app other than adding to them.